x-pack/plugins/maps/public/classes/sources/esql_source/esql_editor.tsx

```diff
@@ -98,6 +98,7 @@ export function ESQLEditor(props: Props) {
         hideRunQueryText
         isLoading={isLoading}
         disableSubmitAction={isEqual(localQuery, props.esql)}
+        hideQueryHistory
```

Comment:
ℹ️ Maps is a unique case as it errors out if the query doesnt return geo fields. the query might be valid though. Marking it as failed would be confusing to the other editor consumers where this query makes sense.
So I am hiding this for now.